lib/ricohapi/mstorage/client.rb

```diff
@@ -90,7 +92,17 @@ def delete(media_id)
 
       # PUT /media/{id}/meta/user/{key}
       def add_meta(media_id, user_meta)
-        # TODO: do something
+        raise Error.new("user_meta is empty: nothing to request.") if user_meta.empty?
```

Comment:
I think "'user metadata' is empty: nothing to request." is better. How do you think ?
Because users don't know what `user_meta` is.
We don't publish the information.> https://github.com/okunoyuki/media-storage-rb/tree/ee54288f86c94e33081fea87abffa2afe20ab894#attach-media-metadata